Shared/Samples/Densify and generalize geometry/DensifyAndGeneralizeGeometryView.swift

```diff
@@ -0,0 +1,188 @@
+        let resultPointsGraphic: Graphic = {
+            let symbol = SimpleMarkerSymbol(style: .circle, color: .magenta, size: 7)
+            return Graphic(symbol: symbol)
+        }()
+        
+        /// The graphic for displaying the lines of the resultant geometry.
+        let resultPolylineGraphic: Graphic = {
+            let symbol = SimpleLineSymbol(style: .solid, color: .magenta, width: 3)
+            return Graphic(symbol: symbol)
+        }()
+        
+        /// A mutable point collection from which the original polyline and
+        /// multipoint geometries are made.
+        private let pointCollection: MutablePointCollection = {
+            // The spatial reference for the sample.
+            let spatialReference = SpatialReference(wkid: WKID(rawValue: 32126)!)
+            var collection = MutablePointCollection(
+                points: [
+                    Point(x: 2330611.130549, y: 202360.002957),
+                    Point(x: 2330583.834672, y: 202525.984012),
+                    Point(x: 2330574.164902, y: 202691.488009),
+                    Point(x: 2330689.292623, y: 203170.045888),
+                    Point(x: 2330696.773344, y: 203317.495798),
+                    Point(x: 2330691.419723, y: 203380.917080),
+                    Point(x: 2330435.065296, y: 203816.662457),
+                    Point(x: 2330369.500800, y: 204329.861789),
+                    Point(x: 2330400.929891, y: 204712.129673),
+                    Point(x: 2330484.300447, y: 204927.797132),
+                    Point(x: 2330514.469919, y: 205000.792463),
+                    Point(x: 2330638.099138, y: 205271.601116),
+                    Point(x: 2330725.315888, y: 205631.231308),
+                    Point(x: 2330755.640702, y: 206433.354860),
+                    Point(x: 2330680.644719, y: 206660.240923),
+                    Point(x: 2330386.957926, y: 207340.947204),
+                    Point(x: 2330485.861737, y: 207742.298501)
+                ],
+                spatialReference: spatialReference)
+            return collection
+        }()
+        
+        /// A Boolean indicating whether to generalize.
+        @Published var shouldGeneralize = false
+        
+        /// The max deviation for generalization.
+        @Published var maxDeviation = 10.0
+        
+        /// A Boolean indicating whether to densify.
+        @Published var shouldDensify = false
+        
+        /// The max segment length for densifying.
+        @Published var maxSegmentLength = 100.0
+        
+        init() {
+            originalPolyline = Polyline(points: pointCollection)
+            
+            // Create map.
+            map = Map(basemapStyle: .arcGISStreetsNight)
+            
+            // Set the initial viewpoint to show the extent of the graphics.
+            map.initialViewpoint = Viewpoint(
+                center: originalPolyline.extent.center,
+                scale: 65907
+            )
```

Comment:
I'm not sure what the sample coding standards are, but this could happen as part of the declaration of the `map` variable if you wanted it to.

something like:

```swift
let map = {
    ...
    return map
}()
```

Reply:
I had it done in the constructor since you can't set the initial viewpoint in a property initializer because you use the original polyline to do so and the property initializers run before 'self' is available. I moved map creation up there though and kept the setting of the initial viewpoint in the constructor.